mojio_sdk/battery.py

```diff
@@ -1,7 +1,14 @@
 class Battery:
 
     def __init__(self, json_data):
+        self.is_empty = False
```

Comment:
An attribute `is_empty` can be confusing when used in context with a battery. Maybe you can think of a better name? Like `has_data` (and then inverse it compared to the current approach of course)?